packages/format-library/src/text-color/inline.js

```diff
@@ -142,6 +142,7 @@ function ColorPicker( { name, property, value, onChange } ) {
 					setColors( value, name, colors, { [ property ]: color } )
 				);
 			} }
+			__experimentalIsRenderedInSidebar
```

Comment:
This prop could use better naming, not specific to the location where the component is rendered.

Do you mind adding a small inline comment explaining what it does in this particular case?

Reply:
Thank you for the review!
Of course. Thank you.